ldes-server-infra-mongo/mongo-repository/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/infra/mongo/member/service/LdesMemberEntityListener.java

```diff
@@ -0,0 +1,25 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.infra.mongo.member.service;
+
+import be.vlaanderen.informatievlaanderen.ldes.server.infra.mongo.member.entity.LdesMemberEntity;
+import be.vlaanderen.informatievlaanderen.ldes.server.infra.mongo.membersequence.service.SequenceGeneratorService;
+import org.springframework.data.mongodb.core.mapping.event.AbstractMongoEventListener;
+import org.springframework.data.mongodb.core.mapping.event.BeforeConvertEvent;
+import org.springframework.stereotype.Component;
+
+@Component
+public class LdesMemberEntityListener extends AbstractMongoEventListener<LdesMemberEntity> {
```

Comment:
This name is quite generic for a very specific listener, however I can't really come up with a better name without making it super long like LdesMemberEntityMongoEventListener :(

Reply:
quote is on variables, but I guess it also applies to classes :-)

“You should name a variable using the same care with which you name a first-born child.”

― Robert C. Martin, [Clean Code: A Handbook of Agile Software Craftsmanship]